docs/admin/reviewing_logs.rst

```diff
@@ -0,0 +1,42 @@
+Reviewing and exporting logs
```

Comment:
This discusses log export, not review - it might make sense to consolidate it into the clipboard management page instead, giving the latter a structure like:
- risks and benefits
- example: enabling one-way copy/paste from the `vault` VM to `sd-app`
- example: enabling two-way copy/paste between `sd-app` and a user-managed VM
- example: exporting logs from `sd-logs`.

Reply:
There are two main issues I have with that:
- it makes the documentation less task-centric and more developer-centric
- it conflates risks and benefits of two very different RPC policy changes.

On the first point, as an admin, my understanding of the system may not extend to RPC policies. I likely have a very specific goal: enable copy and paste, get at the logs, etc. Right now it's very easy to find that specific _goal_ in the table of contents.

On the second point, the permission to copy files from a VM that only collects logs to another VM is very different in terms of its security risks from the clipboard-related permissions.

That said, let me look at the overall shape of the docs once we move stuff around in the install section (will need to test that procedure as well).

Reply:
I still prefer the current organization for these reasons, but as with the password management, if you feel strongly differently, feel free to push a commit to reorganize and I can review. I would just ask that we try to make it as easy as possible for an admin to locate the specific instructions they need to complete the task they're interested in.

Reply:
Exfiltration risk via logs or clipboard are pretty similar; but again, due to release timing this can go as-is with a potential follow-up PR.